docs/manuals/userman/gui/viselements/generic/progress.md_template

```diff
@@ -12,4 +12,22 @@ name to select the date selectors on your page and apply style.
 
 # Usage
 
-TODO
+## Circular Progress
+
+The progress render, which is set by default to True, simply handles that if the progress element should be displayed or not.
+
+!!! taipy-element
```

Comment:
This is useless: *render* is not really a feature of the progress bar, and there's no *value* here, which is mandatory.